src/main/java/heronarts/lx/Tempo.java

```diff
@@ -191,16 +194,38 @@ public Tempo(LX lx) {
     addParameter("trigger", this.trigger);
     addParameter("enabled", this.enabled);
     addModulator("nudge", this.nudge);
+
+    // set initial minBpm, maxBpm
+    minBpm = DEFAULT_MIN_BPM;
+    maxBpm = DEFAULT_MAX_BPM;
   }
 
   private static final String PATH_BEAT = "beat";
   private static final String PATH_SET_BPM = "setBPM";
+  
+  public boolean setBpmRange(double min, double max) {
+    if (min <= 0.0 || min >= max) {
+      // do not set to invalid range of BPMs
```

Comment:
I would actually `throw new IllegalArgumentException` here - caller probably not checking for boolean return value here, and there shouldn't be any scenario in which passing invalid arguments to this method is sensible.

Can make the method void, or set it's return type to `Tempo` and have it `return this;` for method chaining.